lib/Migration/Version000009Date20180202213017.php

```diff
@@ -42,16 +42,16 @@ class Version009000Date20180202213017 extends SimpleMigrationStep {
 	public function changeSchema(IOutput $output, \Closure $schemaClosure, array $options) {
 		/** @var ISchemaWrapper $schema */
 		$schema = $schemaClosure();
-		if ($schema->hasTable('polls_dts')) {
+		if ($schema->hasTable('polls_dts') & $schema->hasTable('polls_options')) {
```

Comment:
You mean &&
And I would not check the second table here

Reply:
Yes. I meant &&. Thanks. I thought it would be a good idea to check the existing of the new tables before deleting the old one. Just to prevent data loss. 

But maybe you are right. Migration seems to work properly here.